google/cloud/bigtable/table.py

```diff
@@ -1071,10 +1087,11 @@ def _do_mutate_retryable_rows(self):
         if self.timeout is not None:
             kwargs["timeout"] = timeout.ExponentialTimeout(deadline=self.timeout)
 
+        # todo confirm this change
```

Comment:
@crwilcox another todo I missed - I think my concern here was the mutate rows call wasn't wrapped in a `request={...` but I think due to the code below this (will need to expand) this works as expected. LMK if you have a concern